clients/ts/FunctionalTests/tsconfig.json

```diff
@@ -1,27 +1,12 @@
 {
-  "compileOnSave": false,
+  "extends": "../tsconfig.base.json",
```

Comment:
We now share all compilerOptions from the common `tsconfig.base.json` in `clients\ts`, and only override the things we have to in the individual folders.